.github/pull_request_template.md

```diff
@@ -1,9 +1,26 @@
-<!-- Before submitting a Pull Request, please ensure you have signed the CLA using this GitHub App: https://cla-assistant.io/renovatebot/renovate -->
+<!-- If this is your first pull request: sign the CLA with this GitHub app: https://cla-assistant.io/renovatebot/renovate -->
+<!-- Make sure the `Allow edits and access to secrets by maintainers` checkbox is checked on this pull request. -->
+<!-- Your pull request title must be a Conventional Commits style message (https://www.conventionalcommits.org/en/v1.0.0/) -->
+<!-- If your pull request is going to change a lot of things, open an issue for discussion before starting your work. -->
 
-<!-- When creating this PR on GitHub, please ensure `Allow edits from maintainers.` checkbox is checked -->
+## Changes:
 
-<!-- Please use a conventional commit message (https://www.conventionalcommits.org/en/v1.0.0/) as your PR title -->
+<!-- Describe what this pull request changes. -->
 
-<!-- Ideally include at least one sentence saying what this PR achieves -->
+## Context:
 
-<!-- Finish the PR with `Closes #` and then the issue number if it closes any associated issue -->
+<!-- Describe why you're making these changes. -->
+<!-- If you're closing an existing issue with this pull request, use the keyboard Closes #issue_number -->
```

Comment:
```suggestion
<!-- If you're closing an existing issue with this pull request, use the keyword Closes #issue_number -->
```

Reply:
Good catch, I don't know why I typed that.... :smile: